src/radical/pilot/agent/scheduler/continuous.py

```diff
@@ -383,96 +426,121 @@ def _alloc_mpi(self, cud):
         # requested_cores > cores_per_node on impossible full-node-chunking
 
         cores_per_node = self._lrms_cores_per_node
-        gpus_per_node  = self._lrms_gpus_per_node
+        gpus_per_node = self._lrms_gpus_per_node
+        lfs_per_node = self._lrms_lfs_per_node   # this is a dictionary
+        # which have two key,s size (MB) and path
 
```

Comment:
Why is this explained here, but not on other uses?  Should probably be explained where its being defined, not when being used?